hnn_core/network.py

```diff
@@ -140,33 +142,40 @@ class Network(object):
     ----------
     params : dict
         The parameters to use for constructing the network.
+    initialise_hnn_drives : bool
+        If True (default), add drives as in HNN-GUI (two global ("common")
+        bursty drives providing rhythmic drive, one distal and two proximal
+        evoked drives, one Gaussian and one Poisson background drive).
 
     Attributes
     ----------
     params : dict
         The parameters of the network
     cellname_list : list
         The names of real cell types in the network (e.g. 'L2_basket')
-    feedname_list : list
-        The names of external drivers ('feeds') to the network (e.g. 'evdist1')
     gid_ranges : dict
         A dictionary of unique identifiers of each real and artificial cell
         in the network. Every cell type is represented by a key read from
-        cellname_list, followed by entries read from feedname_list. The value
+        cellname_list, followed by keys read from external_drives. The value
         of each key is a range of ints, one for each cell in given category.
         Examples: 'L2_basket': range(0, 270), 'evdist1': range(272, 542), etc
     pos_dict : dict
         Dictionary containing the coordinate positions of all cells.
         Keys are 'L2_pyramidal', 'L5_pyramidal', 'L2_basket', 'L5_basket',
-        'common', or any of the elements of the cellname or feedname lists.
+        or any external drive name
     cell_response : CellResponse
         An instance of the CellResponse object.
-    feed_times : dict of [list (n_trials) of list (n_cells) of list (n_times)]
-        The event times of input feeds. Only feed_times['tonic'] is a
-        dictionary, other input types are nested lists.
+    external_drives : dict of NetworkDrive
+        The external driving inputs to the network. Drives are added by
+        defining their spike-time dynamics, and their connectivity to the real
+        cells of the network. Event times are instantiated before simulation,
+        and are stored in each NetworkDrive['events'] (list of list; first
+        index for trials, second for event time lists for each drive cell).
```

Comment:
I think you should document all the possible keys of `NetworkDrive` in the class description. So we keep it under control and not let it grow arbitrarily